server/blob/blob-objectstorage/pom.xml

```diff
@@ -85,6 +85,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
+            <version>25.1-jre</version>
```

Comment:
That's cheating! 

We need a bit of analysis here: **which dependency is pulling and relying on guava 25 dependency?** -> jcloud-core does...

Reply:
that was the jclouds dependency, we encountered this problem before and I saw that was what we did: overriding guava version. (mpt/impl/imap-mailbox/cyrus/pom.xml and mailbox/zoo-seq-provider/pom.xml).

Reply:
These are not used in any "final" guice product - hence this solution is not applicable

Reply:
okay, I put the guava upgrade into an separated PR https://github.com/linagora/james-project/pull/2580. And try to find a way